client/app/bundles/shared/components/Checkbox.jsx

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React from "react";
+import css from "./Checkbox.scss";
+
+type Props = {
+  id: string,
+  value?: string | number,
+};
+
+type State = {
+  checkboxState: boolean;
+};
+
+export default class Checkbox extends React.Component<Props, State>{
+  constructor(props: Props) {
+    super(props);
+    this.state = { checkboxState: true};
+  }
+
+  toggle = () => {
+    this.setState({ checkboxState: !this.state.checkboxState });
+  }
+
+  render() {
+    const checkedOrNot = [];
+    checkedOrNot.push(
+      <p>{this.state.checkboxState ? 'Unchecked' : 'Checked'}</p>
```

Comment:
Unchecked and Checked should be i18n keys

Reply:
I don't know how to do this. Can you help me figure it out .